README.md

```diff
@@ -23,6 +23,27 @@ ESLint configuration is provided in the `eslint.config.js`, aka. "Flat Config" f
    ```
    This will set up the necessary dependencies and configurations for you.
 
+## Migration von v2 to v3
+
+When upgrading from `@atmina/linting@^2` to `@^3` in a Next project, a few changes are currently required:
+1. Add `@eslint/compat` as a dev-dependency to the Next project
+2. Update your `eslint.config.js` by wrapping the next plugin's config with `fixupConfigRules`:
+```diff
+// For CJS
++ const {fixupConfigRules} = require('@eslint/compat');
+// ...
+- require('@atmina/linting/eslint/next')(require('@next/eslint-plugin-next')),
++ ...fixupConfigRules(
```

Comment:
Is there a reason we can't do this internally in `linting/eslint/next`? That way we wouldn't need to change the `eslint.config.js` at all.

Reply:
Very valid point! I moved the wrap with `fixupConfigRules` into our config. `fixupConfigRules` always returns an array but since we only pass one object in, it's safe to return the first element. This avoids having to spread the result of the fixed config in our projects.